src/directory/directory.js

```diff
@@ -647,17 +647,47 @@ const directory = {
           {
             title: 'Getting started',
             route: '/lib/push-notifications/getting-started',
-            filters: ['react-native']
+            filters: ['ios', 'react-native']
+          },
+          {
+            title: 'Register device',
+            route: '/lib/push-notifications/register-device',
+            filters: ['ios']
+          },
+          {
+            title: 'Record notifications',
+            route: '/lib/push-notifications/record-notifications',
+            filters: ['ios']
+          },
+          {
+            title: 'Identify user',
+            route: '/lib/push-notifications/identify-user',
+            filters: ['ios']
           },
           {
             title: 'Working with API',
             route: '/lib/push-notifications/working-with-api',
             filters: ['react-native']
           },
+          {
+            title: 'Remote media',
+            route: '/lib/push-notifications/remote-media',
+            filters: ['ios']
+          },
           {
             title: 'Testing',
             route: '/lib/push-notifications/testing',
-            filters: ['react-native']
+            filters: ['ios', 'react-native']
+          },
+          {
+            title: 'Setting up push notification services',
```

Comment:
shouldn't this page be right after getting started?

Reply:
it's really a prerequisite to getting started and linked from the prerequisite section, so when the reader finishes the getting started page it should already be done.  If we put it next on the list, it is what's linked from the `next` button at the bottom of the getting started page, which is a bad user experience.  I put it at the end so it's still easy to get to for reference, but not part of the getting started flow.  This is the same way it was handled in the existing sdk documentation.